src/datasources/system/SystemDataSource.ts

```diff
@@ -24,15 +24,26 @@ export class SystemDataSource extends DataSourceApi<SystemQuery> {
     const to = range!.to.valueOf();
 
     // Return a constant for each query.
-    const data = options.targets.map((target) => {
-      return new MutableDataFrame({
-        refId: target.refId,
-        fields: [
-          { name: 'Time', values: [from, to], type: FieldType.time },
-          { name: 'Value', values: [target.constant, target.constant], type: FieldType.number },
-        ],
-      });
-    });
+    const data = await Promise.all(options.targets.map(async (target) => {
+      if (target.queryClass === QueryType.Summary) {
+        var summaryResponse = await getBackendSrv().get<SystemSummary>(this.baseUrl + '/get-systems-summary');
+        return new MutableDataFrame({
+          refId: target.refId,
+          fields: [
+            { name: 'Connected Count', values: [summaryResponse.connectedCount], type: FieldType.number },
+            { name: 'Disconnected Count', values: [summaryResponse.disconnectedCount], type: FieldType.number },
+          ],
+        });
+      } else {
+        return new MutableDataFrame({
+          refId: target.refId,
+          fields: [
+            { name: 'Time', values: [from, to], type: FieldType.time },
+            { name: 'Value', values: [1, 2], type: FieldType.number },
```

Comment:
What are the hardcoded 1 and 2 representative of? Can we pull these out as constants with a descriptive name?

Reply:
Oh everything in the else statement is just a placeholder from the template. I can change it to return null for now if that's better?

Reply:
If the placeholder values truly serve no purpose, then yes I'd suggest we don't set a value instead.